src/types.ts

```diff
@@ -1586,31 +1777,139 @@ export interface KAPLAYCtx<
         button: MouseButton | MouseButton[],
         action: (m: MouseButton) => void,
     ): KEventController;
+    /**
+     * Register an event that runs every frame when any mouse button is being held down.
+     * 
+     * @example
+     * ```js
+     * // count time with any mouse button down
+     * let mouseTime = 0
+     * onMouseDown((m) => {
+     *     mouseTime += dt()
+     *     debug.log(`Time with mouse down: ${mouseTime})
+     *     debug.log(`Mouse button down: ${m})
+     * })
+     * ```
+     *
+     * @since v3001.0
+     * @group Input
+     */
     onMouseDown(action: (m: MouseButton) => void): KEventController;
     /**
      * Register an event that runs when user clicks mouse.
+     * 
+     * @example
+     * ```js
+     * // gives cookies on left press, remove on right press
+     * let cookies = 0
+     * onMousePress([`left`, `right`], (m) => {
+     *     if (m == `left`)
+     *         cookies++
```

Comment:
please use our format with brackets or change to inline `if(m == "left") cookies++;`